configs/testnet.json

```diff
@@ -85,5 +85,27 @@
     "reward_pool": 1000,
     "start": 1742284800,
     "end": 1742374800
+  },
+  "whitelisted_rpc_urls": {
+    "Arbitrum": "https://rinkeby.arbitrum.io/rpc",
+    "Avalanche": "https://ava-testnet.public.blastapi.io/ext/bc/C/rpc",
+    "Base": "https://base-sepolia-rpc.publicnode.com",
+    "Binance Smart Chain": "https://data-seed-prebsc-1-s2.binance.org:8545/",
+    "Ethereum": "https://sepolia.drpc.org",
+    "Mantle": "https://rpc.sepolia.mantle.xyz",
+    "OKC": "https://exchaintestrpc.okex.org",
+    "Optimism": "https://sepolia.optimism.io",
+    "Polygon": "https://rpc-mumbai.maticvigil.com"
+  },
+  "chain_name_to_evm_id_map": {
+    "Arbitrum": 421611,
+    "Avalanche": 43113,
+    "Binance Smart Chain": 97,
+    "Ethereum": 5,
+    "Mantle": 5003,
+    "OKC": 65,
+    "Optimism": 11155420,
+    "Polygon": 80001,
+    "Base": 84532
```

Comment:
Pls make sure the blockchain names match those from the Bridge Connections [API](https://github.com/Switcheo/demex-webapp-config/pull/350/files#r2007116808)
![Screenshot 2025-03-27 111718](https://github.com/user-attachments/assets/b21d5692-8260-43a7-8c95-09cffc191e23)